### ISSUE TITLE
Add new age group and output for federal states to getVaccinationData

### ISSUE THREAD

**Issue by mknaranja** [DLR-SC/memilio#166]
Since Dec 21, 2021, new vaccination data for age group 5-11 is available. This has to be added.

Also output for federal states aggregation should be possible.

**Comment by mknaranja** [comment]
I probably found the error in the vaccination script. A "lower" had to be replaced by "lower equal". What I'd checked:

- Data frames with moving average = False or 7 for lower and lower equal are identical for data of December 20, 2021
- Number of total complete or refreshed vaccinations is identical for lower equal for input data frame and transformed data frames of January 16, 2022:
```
df_data[df_data.Impfschutz==2].Number.sum()-df_data_agevacc_county_cs.loc[df_data_agevacc_county_cs.Date=="2022-01-16", "Vacc_completed"].sum()
0.0
df_data[df_data.Impfschutz==3].Number.sum()-df_data_county_cs.loc[df_data_county_cs.Date=="2022-01-16", "Vacc_refreshed"].sum()
0.0
df_data[df_data.Impfschutz==2].Number.sum()-df_data_ageinf_county_cs.loc[df_data_ageinf_county_cs.Date=="2022-01-16", "Vacc_completed"].sum()
0.0
```
- Number of vaccinations in age group 05-11 is larger than zero:
```
df_data_agevacc_county_cs.loc[(df_data_agevacc_county_cs.Date=="2022-01-16") & (df_data_agevacc_county_cs.Age_RKI=="05-11"),"Vacc_partially"].sum()
10127.0
```

**Comment by annette-lutz** [comment]
Closed by #177
